src/Character/Camera/DefaultCamera.lua

```diff
@@ -42,6 +42,10 @@ function DefaultCamera.ShouldHidePart(Part: BasePart): boolean
         end
     end
 
+    if Part:FindFirstAncestorOfClass("Tool") then
+		return false
+	end
```

Comment:
The formatting of these 2 lines is tabs instead of 4 spaces. This may look fine in Roblox Studio and other editors, but it looks wrong on GitHub.